requirements-python.txt

```diff
@@ -1,35 +1,35 @@
-PyYAML==5.1
-paramiko==2.7.1
+PyYAML==5.3.1
+paramiko==2.7.2
 fabric==2.5.0
 invoke==1.3.1
-boto3==1.15.3
-boto3-stubs[s3,ec2,dynamodb,pricing]==1.15.3.0
-awscli==1.18.153
+boto3==1.15.16
+boto3-stubs[s3,ec2,dynamodb,pricing]==1.15.16.0
+awscli==1.18.157
 aexpect==1.2.0
 cassandra-driver==3.22.0
 apache-libcloud==2.6.0
 requests==2.20.0
-elasticsearch==6.0.0
+elasticsearch==6.8.1
 sortedcontainers==1.5.9
 jinja2==2.10.1
-prometheus_client==0.3.1
+prometheus_client==0.8.0
 pre-commit==1.14.4
 anyconfig==0.9.8
 click==7.0
 click-completion==0.5.0
 PTable==0.9.2
-pycryptodome==3.8.1
-pyzmq==18.0.1
+pycryptodome==3.9.8
+pyzmq==19.0.2
 tenacity==5.0.4
 # this package isn't in pypi
 https://github.com/fruch/repodataParser/archive/py3.zip ; python_version > '3'
 pytest==4.6.4
-tcconfig==0.23.0
+tcconfig==0.26.0
 selenium==3.141.0
 mysql-connector-python==8.0.19
 docker==4.2.0
 python-jenkins==1.4.0
-ssh2-python==0.18.0.post1
+ssh2-python==0.22.0
```

Comment:
when they did released this one ?

Reply:
They released 0.20.0, 0.21.0 and 0.22.0 in last three weeks. But major change was 0.19.0 in mid-August.